Spec/ObjectLifetimes.swift

```diff
@@ -1,143 +1,136 @@
 import Ably
-import Quick
 import Nimble
-            private let options: ARTClientOptions = {
-                let options = ARTClientOptions(key: "fake:key")
-                options.autoConnect = false
-                return options
-            }()
+import Quick
+private let options: ARTClientOptions = {
```

Comment:
TODO there's no breathing space here

Reply:
I've updated the migrator to add a newline before the first global declaration that it inserts.